netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -39,6 +49,9 @@ public sealed partial class ConstructorBuilder : System.Reflection.ConstructorIn
         public override System.Reflection.Module Module { get { throw null; } }
         public override string Name { get { throw null; } }
         public override System.Type ReflectedType { get { throw null; } }
+//      [System.ObsoleteAttribute("This property has been deprecated. http://go.microsoft.com/fwlink/?linkid=14202")]
+//      public System.Type ReturnType { get { throw null; } }
+//CAS   public void AddDeclarativeSecurity(System.Security.Permissions.SecurityAction action, System.Security.PermissionSet pset) { }
```

Comment:
I assume "//CAS" is shorthand for "// Excluded because we don't support Code Access Security"? It might be simpler to keep the consistent format across reasons.

Reply:
Yes, we're using CAS all over the place already. At least on our end that term is well understood.